spring-pulsar/build.gradle

```diff
@@ -25,8 +25,7 @@ dependencies {
 	optional 'io.projectreactor:reactor-core'
 	testImplementation 'org.junit.jupiter:junit-jupiter'
 	testRuntimeOnly 'org.junit.platform:junit-platform-launcher'
-	testRuntimeOnly 'org.apache.logging.log4j:log4j-core'
-	testRuntimeOnly 'org.apache.logging.log4j:log4j-jcl'
+	testRuntimeOnly("ch.qos.logback:logback-classic:1.4.1")
```

Comment:
The tests were not respecting the logback-test.xml file as it was using log4j for tests. This falls in line w/ the default logging choice when using the SB starter (it uses logback by default as well).